.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
I think if we add this file to the repo, we shoud filll it properly with updated information, or not add it at all

Reply:
Ok !